src/status_im2/contexts/activity_center/notification/membership/view.cljs

```diff
@@ -1,45 +1,84 @@
-   [quo/activity-log
-    {:title     (i18n/label :t/added-to-group-chat)
-     :icon      :i/add-user
-     :timestamp (datetime/timestamp->relative timestamp)
-     :unread?   (not read)
-     :context   [[common/user-avatar-tag author]
-                 (i18n/label :t/added-you-to)
-                 [quo/group-avatar-tag chat-name
-                  {:size  :small
-                   :color :purple}]]
-     :items     (when-not accepted
-                  [{:type                :button
-                    :subtype             :positive
-                    :key                 :button-accept
-                    :label               (i18n/label :t/accept)
-                    :accessibility-label :accept-group-chat-invitation
-                    :on-press            #(rf/dispatch [:activity-center.notifications/accept id])}
-                   {:type                :button
-                    :subtype             :danger
-                    :key                 :button-decline
-                    :label               (i18n/label :t/decline)
-                    :accessibility-label :decline-group-chat-invitation
-                    :on-press            #(rf/dispatch [:activity-center.notifications/dismiss
-                                                        id])}])}]])
+  [{:keys [notification set-swipeable-height] :as props}]
+  (let [{:keys [id accepted dismissed author read timestamp chat-name chat-id]} notification]
+    [swipeable props
+     [pressable {:accepted accepted :chat-id chat-id}
+      [quo/activity-log
+       {:title     (i18n/label :t/added-to-group-chat)
+        :on-layout set-swipeable-height
+        :icon      :i/add-user
+        :timestamp (datetime/timestamp->relative timestamp)
+        :unread?   (not read)
+        :context   [[common/user-avatar-tag author]
+                    (i18n/label :t/added-you-to)
+                    [quo/group-avatar-tag chat-name
+                     {:size  :small
+                      :color :purple}]]
+        :items     (when-not (or accepted dismissed)
+                     [{:type                :button
+                       :subtype             :positive
+                       :key                 :button-accept
+                       :label               (i18n/label :t/accept)
+                       :accessibility-label :accept-group-chat-invitation
+                       :on-press            #(rf/dispatch [:activity-center.notifications/accept id])}
+                      {:type                :button
+                       :subtype             :danger
+                       :key                 :button-decline
+                       :label               (i18n/label :t/decline)
+                       :accessibility-label :decline-group-chat-invitation
+                       :on-press            #(rf/dispatch [:activity-center.notifications/dismiss
+                                                           id])}])}]]]))
```

Comment:
not sure if this is considered dangling

Reply:
Ugly right? I would never format Clojure code like this, but due to the max line length, `zprint` is adding a line break. Unfortunately this is a common formatting side-effect in the codebase, I see tons of examples. I can live with that, but it is indeed ugly... Dangling would be lonely parentheses, brackets, etc on new lines.